src/ProjectTemplates/GeneratedContent.targets

```diff
@@ -0,0 +1,72 @@
+<Project>
+
+  <PropertyGroup>
+    <_ChatWithCustomDataWebContentRoot>$(MSBuildThisFileDirectory)Microsoft.Extensions.AI.Templates\src\ChatWithCustomData\ChatWithCustomData.Web-CSharp\</_ChatWithCustomDataWebContentRoot>
+  </PropertyGroup>
+
+  <Target Name="ComputeGeneratedContentProperties">
+    <PropertyGroup>
+      <!-- Define optional pinned versions of certain dependencies. -->
+      <TemplatePinnedMicrosoftExtensionsAIVersion>9.3.0-preview.1.25114.11</TemplatePinnedMicrosoftExtensionsAIVersion>
+      <TemplatePinnedMicrosoftEntityFrameworkCoreSqliteVersion>9.0.2</TemplatePinnedMicrosoftEntityFrameworkCoreSqliteVersion>
+
+      <!-- By default, don't use pinned dependency versions. -->
+      <TemplateUsePinnedMicrosoftExtensionsAIVersion Condition="'$(TemplateUsePinnedMicrosoftExtensionsAIVersion)' == ''">false</TemplateUsePinnedMicrosoftExtensionsAIVersion>
+      <TemplateUsePinnedMicrosoftEntityFrameworkCoreSqliteVersion Condition="'$(TemplateUsePinnedMicrosoftEntityFrameworkCoreSqliteVersion)' == ''">false</TemplateUsePinnedMicrosoftEntityFrameworkCoreSqliteVersion>
+
+      <!-- Use pinned dependency versions if specified. -->
+      <TemplateMicrosoftExtensionsAIVersion Condition="'$(TemplateUsePinnedMicrosoftExtensionsAIVersion)' == 'true'">$(TemplatePinnedMicrosoftExtensionsAIVersion)</TemplateMicrosoftExtensionsAIVersion>
+      <TemplateMicrosoftEntityFrameworkCoreSqliteVersion Condition="'$(TemplateUsePinnedMicrosoftEntityFrameworkCoreSqliteVersion)' == 'true'">$(TemplatePinnedMicrosoftEntityFrameworkCoreSqliteVersion)</TemplateMicrosoftEntityFrameworkCoreSqliteVersion>
+
+      <!-- Fall back on default dependency versions. -->
+      <TemplateMicrosoftExtensionsAIVersion Condition="'$(TemplateMicrosoftExtensionsAIVersion)' == ''">$(Version)</TemplateMicrosoftExtensionsAIVersion>
+      <TemplateMicrosoftEntityFrameworkCoreSqliteVersion Condition="'$(TemplateMicrosoftEntityFrameworkCoreSqliteVersion)' == ''">$(MicrosoftEntityFrameworkCoreSqliteVersion)</TemplateMicrosoftEntityFrameworkCoreSqliteVersion>
+
+      <_TemplateUsingJustBuiltPackages Condition="'$(TemplateMicrosoftExtensionsAIVersion)' == '$(Version)'">true</_TemplateUsingJustBuiltPackages>
+
+      <!-- Specify package version variables used in template content. -->
+      <GeneratedContentProperties>
+        $(GeneratedContentProperties);
+
+        <!-- Repo properties -->
+        ArtifactsShippingPackagesDir=$(ArtifactsShippingPackagesDir);
+
+        <!-- Package version properties -->
+        OllamaSharpVersion=$(OllamaSharpVersion);
```

Comment:
Where is this pulled from? Presumably it'll need to be updated to whatever version of OllamaSharp includes https://github.com/awaescher/OllamaSharp/pull/200

Reply:
Just updated to the latest version. Validating now that everything works.

Reply:
Does the latest OllamaSharp allow is to eliminate the streaming/nonstreaming distinctions? As in, it works with streaming function calls now, doesn't it?

Reply:
I tried to remove this special-cashing logic, and it was not working. Every response I got back had text of `""`.